doc/vector/riscv-crypto-vector-element-groups.adoc

```diff
@@ -15,15 +15,15 @@ that operates on them:
 - Effective Element Width (EEW) - number of bits in each element
 
 When the EEW is explicitly defined for a vector crypto instruction, the current `SEW` is ignored by that instruction.
-This was done to enable other SEW values to be used without requiring an additional `vsetvl` instruction.
```

Comment:
"vsetvl" is intended to imply all variants. perhaps "vsetlv-type" would be better.

Reply:
I like `vsetvl-type`, I can incorporate it into a future PR.